### DISCUSSION TITLE
[잡담] 오늘 저녁에 뭐 먹을까요...?

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#24] [잡담]
추천 좀...

**Comment by dev-LeoKang** [comment]
저는 오늘 저녁 사케동 먹고 헤네시 마시는데 괜찮네요

**Comment by ghost** [comment]
제가 알바하는 음식점인데 코코이찌방야 추천합니다~~ 카레 전문점인데 카레를 싫어하는 제가 하루도 빠짐없이 카레를 먹게 되어버렸습니다...
